fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ViewDefBuilder.java

```diff
@@ -0,0 +1,327 @@
+            StringBuilder sqlBuilder = new StringBuilder();
+            QueryRelation queryRelation = stmt.getQueryRelation();
+
+            if (queryRelation.hasWithClause()) {
+                sqlBuilder.append("WITH ");
+                List<String> cteStrings =
+                        queryRelation.getCteRelations().stream().map(this::visit).collect(Collectors.toList());
+                sqlBuilder.append(Joiner.on(",").join(cteStrings));
+            }
+
+            sqlBuilder.append(visit(queryRelation));
+
+            if (queryRelation.hasOrderByClause()) {
+                List<OrderByElement> sortClause = queryRelation.getOrderBy();
+                sqlBuilder.append(" order by ");
+                for (int i = 0; i < sortClause.size(); ++i) {
+                    sqlBuilder.append(visit(sortClause.get(i).getExpr()));
+                    sqlBuilder.append((sortClause.get(i).getIsAsc()) ? " asc" : " desc");
+                    sqlBuilder.append((i + 1 != sortClause.size()) ? ", " : "");
+                }
+            }
+
+            // Limit clause.
+            if (queryRelation.getLimit() != null) {
+                sqlBuilder.append(queryRelation.getLimit().toSql());
+            }
+            return sqlBuilder.toString();
+        }
+
+        @Override
+        public String visitCTE(CTERelation relation, Void context) {
+            StringBuilder sqlBuilder = new StringBuilder();
+            sqlBuilder.append(relation.getName());
+
+            if (relation.isResolvedInFromClause()) {
+                if (relation.getAliasWithoutNameRewrite() != null) {
+                    sqlBuilder.append(" AS ").append(relation.getAliasWithoutNameRewrite());
+                }
+                return sqlBuilder.toString();
+            }
+
+            if (relation.getColumnOutputNames() != null) {
+                sqlBuilder.append("(").append(Joiner.on(", ").join(relation.getColumnOutputNames())).append(")");
+            }
+            sqlBuilder.append(" AS (").append(visit(new QueryStatement(relation.getCteQuery()))).append(") ");
+            return sqlBuilder.toString();
+        }
+
+        @Override
+        public String visitSelect(SelectRelation stmt, Void context) {
+            StringBuilder sqlBuilder = new StringBuilder();
+            SelectList selectList = stmt.getSelectList();
+            sqlBuilder.append("SELECT ");
+            if (selectList.isDistinct()) {
+                sqlBuilder.append("DISTINCT");
+            }
+
+            List<String> selectListString = new ArrayList<>();
+            for (int i = 0; i < selectList.getItems().size(); ++i) {
+                SelectListItem item = selectList.getItems().get(i);
+                if (item.isStar()) {
+                    List<Expr> outputExpression = SelectAnalyzer.expandStar(item, stmt.getRelation());
+                    for (Expr expr : outputExpression) {
+                        selectListString.add(visit(expr) + " AS `" + toSQL(expr) + "`");
```

Comment:
So many codes like `` + name + ``, how about replace it with an explicit function call `backQuote` ?